webapp/pom.xml

```diff
@@ -63,6 +63,18 @@
     </dependency>
   </dependencies>
 
+  <build>
+    <plugins>
+      <plugin>
+        <artifactId>maven-war-plugin</artifactId>
+        <version>3.3.0</version>
+        <configuration>
+          <packagingExcludes>.gitignore</packagingExcludes>
+        </configuration>
+      </plugin>
+    </plugins>
+  </build>
+
```

Comment:
Werkt het ook als dit in de parent pom staat? dan is het ook gelijk voor de andere war projecten opgelost!

Reply:
Ja, werkt ook. Goed idee inderdaad. Heb het aangepast